2 reports.sql

```diff
@@ -0,0 +1,24 @@
+use PetGroomingDB
+
+--1)We need a list of how many of each pet we currently have.
+select PetCount = count(*), c.PetType
+from client c 
+group by PetType
+
+--2)We need a list of those who have multiple pets, include how many pets they have.
+select c.FirstName , c.LastName, PetCount = count(*)
+from client c 
+group by c.FirstName, c.LastName
+having  count(*) > 1
+
+--3)We need to know our top customer - the customer that is paying us the most.
+select top 1 * 
```

Comment:
The top customer doesn't have to be the customer that is "Biweekly". Also the top customer can be there total from a few animals, you are only checking per animal.